### PR TITLE
feat: Update servicelevel schema

### PR DESCRIPTION
**Pull request by asanuy**
Changes include support to configure SLIs using the GET_FIELD and GET_CDF_COUNT functions. Additionally, it has been introduced a new threshold property that is necessary for CDF.

### CODE REVIEW

File: pkg/servicelevel/types.go [hunk 1]
```diff
@@ -303,6 +319,16 @@ type ServiceLevelResultQuery struct {
 	NRQL NRQL `json:"nrql"`
 }
 
+// User - The `User` object provides general data about the user.
+type User struct {
```
Comment (asanuy) [inline pkg/servicelevel/types.go:323]: This struct is unrelated to the changes on the API. Is this new object expected and harmless?

Comment (sanderblue) [reply pkg/servicelevel/types.go:323]: In this case it's probably harmless. Not quite sure why it was generated and added to the `Actor` struct, but sometimes Tutone might generate an extraneous type to satisfy the GraphQL schema.
